guides/v2.3/rest/tutorials/msi-order-processing/create-shipment.md

```diff
@@ -46,30 +50,21 @@ You must use the `POST V1/shipment` endpoint to create a shipment.
 
 ``` json
 {
-  "entity": {
-  	"billing_address_id": 2,
-    "customer_id": 2,
-    "order_id": 1,
-    "store_id": 2,
-    "total_qty": 45,
   "items": [
     {
-      "name": "Simple Product 1",
-      "price": 5,
-      "product_id": 7,
-      "sku": "sp1",
-      "order_item_id": 1,
+      "order_item_id": 12,
       "qty": 20
     },
     {
-      "name": "Simple Product 2",
-      "price": 10,
-      "product_id": 8,
-      "sku": "sp2",
-      "order_item_id": 2,
+      "order_item_id": 13,
       "qty": 25
     }
-    ],
+  ],
+  "notify": true,
+  "comment": {
+    "comment": "Shipment from Baltimore"
```

Comment:
Ah I love the addition of the comment!

Reply:
Comment is good, but my Spidey sense hates repetition...